playbooks/vagrant-buildbot.yml

```diff
@@ -0,0 +1,6 @@
+- name: Configure instance(s)
+  hosts: all
+  sudo: True
+  roles:
+    - buildbot-master
```

Comment:
Most roles appear to use underscores instead of dashes, but some use dashes. Should I change the dashes to underscores?